tools/wptserve/wptserve/pipes.py

```diff
@@ -478,7 +479,7 @@ def config_replacement(match):
                     "unexpected token type %s (token '%r'), expected ident or arguments" % (ttype, field)
                 )
 
-        assert isinstance(value, (int,) + types.StringTypes), tokens
+        assert isinstance(value, (int, str)), tokens
```

Comment:
Can you make this `six.string_types`? If we're allowing `str` in Py3, we should allow `unicode` in Py2.